pkg/build/builder/common_test.go

```diff
@@ -32,15 +33,12 @@ func TestBuildInfo(t *testing.T) {
 						},
 					},
 				},
-				Revision: &api.SourceRevision{
-					Git: &api.GitSourceRevision{
-						Commit: "1575a90c569a7cc0eea84fbd3304d9df37c9f5ee",
-					},
-				},
 			},
 		},
 	}
-	got := buildInfo(b)
+	sourceInfo := &git.SourceInfo{}
+	sourceInfo.CommitID = "1575a90c569a7cc0eea84fbd3304d9df37c9f5ee"
+	got := buildInfo(b, sourceInfo)
```

Comment:
yeah, i left it out because there's really no expectation that we'd have a revision but no sourceinfo (fundamentally it means something probably went wrong when we tried to interrogate the git repo to find the sourceinfo), but i'll add the test back in.